app/src/main/java/fr/free/nrw/commons/di/CommonsApplicationModule.java

```diff
@@ -220,6 +221,6 @@ public WikidataEditListener provideWikidataEditListener() {
     @Provides
     @Singleton
     public boolean provideIsBetaVariant() {
-        return BuildConfig.FLAVOR.equals("beta");
+        return ConfigUtils.isBetaFlavour();
```

Comment:
Can we not just remove this now?

Reply:
Yes, that should be done. But it was in the central module file so I reverted that to original state just in case it still has any use.